manifest-staging.yml

```diff
@@ -1,8 +1,10 @@
 inherit: manifest.yml
+instances: 4
```

Comment:
I moved it to 4 for my stress tests so it could be like prod. (plus when I had only one instance, the app would crash under my stress tests too.)

I can revert this change for the long term